spec/datadog/appsec/contrib/sinatra/integration_test_spec.rb

```diff
@@ -0,0 +1,321 @@
+# typed: ignore
+
+require 'datadog/tracing/contrib/support/spec_helper'
+require 'rack/test'
+
+require 'securerandom'
+require 'sinatra/base'
+
+begin
+  require 'rack/contrib/json_body_parser'
+rescue LoadError
+  # fallback for old rack-contrib
+  require 'rack/contrib/post_body_content_type_parser'
+end
+
+require 'datadog/tracing'
+require 'datadog/tracing/metadata/ext'
+require 'datadog/tracing/sampling/ext'
+require 'datadog/tracing/contrib/sinatra/ext'
+require 'datadog/tracing/contrib/sinatra/tracer'
+
+require 'datadog/appsec'
+require 'datadog/appsec/contrib/rack/request_middleware'
```

Comment:
Minor: Do we really need all of these requires? Don't `datadog/tracing` and `datadog/appsec` pull the others?

Reply:
Makes sense for integration test specs.